release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -2068,3 +2068,95 @@ was not allowed to use local images. The container image signature policy has
 been updated to always allow images that reference local storage layers
 directly. Now builds can successfully complete if they contain a postCommit
 hook. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1849173[*BZ#1849173*])
+
+
+[[ocp-4-5-4]]
+=== RHBA-2020:3028 - {product-title} 4.5.4 Bug Fix Update
+
+Issued: 2020-07-30
+
+{product-title} release 4.5.4 is now available. The list of packages included
+in the update is documented in the
+link:https://access.redhat.com/errata/RHBA-2020:3028[RHBA-2020:3028] advisory.
+
+
+[[id="ocp-4-5-4-features"]]
```

Comment:
These IDs are not rendering correctly. I believe when using the `id=` version, it should only be accompanied with one bracket:
```
[id="ocp-4-5-4-features"]
```